src/Knp/Menu/Renderer/TwigRenderer.php

```diff
@@ -8,26 +8,15 @@
 
 class TwigRenderer implements RendererInterface
 {
-    private Environment $environment;
-
-    private MatcherInterface $matcher;
-
-    /**
-     * @var array<string, mixed>
-     */
-    private array $defaultOptions;
-
     /**
      * @param array<string, mixed> $defaultOptions
      */
     public function __construct(
-        Environment $environment,
-        string $template,
-        MatcherInterface $matcher,
-        array $defaultOptions = []
+        private Environment      $environment,
```

Comment:
this kind of alignment is a big no-no, providing no practical advantage (and cluttering diffs when you need to change a single line).
I know that PHPStorm is doing it in some cases (and it's a big PITA) but I'm surprised that php-cs-fixer didn't catch it...
Anyway, please remove all superfluous spaces.

Reply:
I don't think php-cs-fixer has a rule yet to forbid aligning promoted properties.

Reply:
It is the default Autoformat from PHPStorm. I changed it.
A rule for that could be provided in an seperat PR.